pages/single-form.jsx

```diff
@@ -91,6 +91,7 @@ var SingleForm = React.createClass({
         </div>
         <SmallPrint stripeNotice={true} />
         <Footer/>
+        <script async src="https://checkout.stripe.com/checkout.js"></script>
```

Comment:
So, it'd be nice to leave this inline so the browser's preloader can start loading this as soon as possible, but it's not required to complete the first render of the page, hence async